saleor/checkout/base_calculations.py

```diff
@@ -7,27 +7,43 @@
 from decimal import Decimal
 from typing import TYPE_CHECKING, Iterable, Optional
 
-from prices import TaxedMoney
+from prices import Money, TaxedMoney
 
 from ..core.prices import quantize_price
-from ..core.taxes import zero_money, zero_taxed_money
+from ..core.taxes import zero_money
 from ..discount import DiscountInfo, VoucherType
 from ..order.interface import OrderTaxedPricesData
-from .fetch import CheckoutLineInfo, ShippingMethodInfo
-from .interface import CheckoutPricesData, CheckoutTaxedPricesData
+from .fetch import CheckoutInfo, CheckoutLineInfo
 
 if TYPE_CHECKING:
     from ..channel.models import Channel
-    from ..checkout.fetch import CheckoutInfo
     from ..order.models import OrderLine
+    from .fetch import ShippingMethodInfo
+
+
+def _calculate_undiscounted_base_line_unit_price(
+    line_info: "CheckoutLineInfo",
+    channel: "Channel",
+):
+    """Calculate line unit price without including discounts and vouchers."""
+    variant = line_info.variant
+    variant_price = variant.get_price(
+        line_info.product,
+        line_info.collections,
+        channel,
+        line_info.channel_listing,
+        [],
+        line_info.line.price_override,
+    )
+    return quantize_price(variant_price, variant_price.currency)
```

Comment:
I will suggest putting this function just under that call of this method to increase the readability.